### PR TITLE
add "help" command to print all commands

### PR DESCRIPTION
**Pull request by l3ifeng**
```
minos # help
tty     - tty related command
clear   - Clear the screec
ps      - List all task information
help    - print command description
vm      - virtual machine cmd
```

### CODE REVIEW

**Comment by l3ifeng** [comment]
@lemin9538 Request review

File: libsys/shell_command/help_cmd.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include <minos/minos.h>
+#include <minos/shell_command.h>
+
+extern unsigned long __shell_command_start;
+extern unsigned long __shell_command_end;
+
+static int
+help_cmd(int argc, char** argv)
```
Comment (lemin9538) [inline libsys/shell_command/help_cmd.c:10]: need change to 

static int help_cmd(int argc, char **argv)

**Comment by l3ifeng** [comment]
@lemin9538 It looks like you don't like Mozilla style, well, working on it.

**Comment by lemin9538** [comment]
@l3ifeng Yes Minos use the linux style :)

**Pull request merged**